docs/data/sdks/browser@2/migration.md

```diff
@@ -0,0 +1,228 @@
+
+#### Using an alternative storage API
+
+```diff
+  amplitude.init(API_KEY, undefined, {
+-   disableCookies: true,
++   identityStorage: 'localStorage',
+  });
+```
+
+#### Disabling user identity persistence
+
+```diff
+- import { MemoryStorage } from '@amplitude/analytics-core';
+-
+  amplitude.init(API_KEY, undefined, {
+-   cookieStorageProvider: new MemoryStorage(),
++   identityStorage: 'none',
+  });
+```
+
+#### Configuring cookie options
+
+The options to manage cookie usage are now nested under `options.cookieOptions` for a more discoverable interface.
+
+```diff
+  amplitude.init(API_KEY, undefined, {
+-   cookieExpiration: 365,
+-   cookieSameSite: 'Lax',
+-   cookieSecure: false,
+-   cookieUpgrade: true,
+-   domain: '',
++   cookieOptions: {
++     expiration: 365,
++     sameSite: 'Lax',
++     secure: false,
++     upgrade: true,
++     domain: '',
++   },
+  });
+```
+
+### No to enums
+
+Amplitude no longer requires the use of enums specifically `TransportType`, `ServerZone` and `PluginType`, and accepts its literaal values.
+
+Setting transport provider on initialization
+
+```diff
+  import * as amplitude from '@amplitude/analytics-browser';
+
+  amplitude.init(API_KEY, USER_ID, {
+-   transport: amplitude.Types.TransportType.Fetch,
++   transport: 'fetch',
+  });
+```
+
+Setting transport provider using setTransport()
+
+```diff
+  import * as amplitude from '@amplitude/analytics-browser';
+
+- amplitude.setTransport(amplitude.Types.TransportProvider.Fetch);
++ amplitude.setTransport('fetch');
+```
+
+Setting server zone on initialization
+
+```diff
+  import * as amplitude from '@amplitude/analytics-browser';
+
+  amplitude.init(API_KEY, USER_ID, {
+-   serverZone: amplitude.Types.ServerZone.US,
++   serverZone: 'US',
+  });
+```
+
+### Simplified plugin interface
+
+Amplitude has made it easier to create your own plugins, requiring less properties for faster authoring.
+
+--8<-- "includes/sdk-ts-browser/shared-plugin-2-properties.md"
+
+### Comparison
+
+This section provides a comparison between different version of Browser SDK. This overview provides key insights for developers, identifying areas that have significant changes and need careful consideration.
+
+
+| Feature | <div class="big-column"> [Browser SDK 2.0](../) </div> | <div class="big-column"> [Browser SDK 1.0](../../typescript-browser/) <div> |
+| --- | --- | --- |
+| Package | [@amplitude/analytics-browser@2](https://www.npmjs.com/package/@amplitude/analytics-browser) | [@amplitude/analytics-browser@1](https://www.npmjs.com/package/@amplitude/analytics-browser) |
+| Web Attribution | Enabled by default with [**Web Attribution V2**](./#web-attribution-v2-vs-web-attribution-v1-vs-maintenance-web-attribution). | Enabled by default with [**Web Attribution V1**](./#web-attribution-v2-vs-web-attribution-v1-vs-maintenance-web-attribution). |
+| Default Event Tracking |  Enabled by default | Disabled by default |
+
+### Web Attribution V2 vs Web Attribution V1
```

Comment:
```suggestion
#### Web Attribution V2 vs Web Attribution V1
```

Reply:
What do you think about the migration doc for MA to Browser 2.0. I feel both MA and browser 1.0 can use the same migration doc.
If that's the case, let's also add the MA to the comparison.